src/icm20948sensor.cpp

```diff
@@ -0,0 +1,895 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2021 Eiren Rain, S.J. Remington
```

Comment:
Are you sure S. J. Remington has any touch in this? His copyright only on I2CDev files, please remove it if it's not his file :D You better place here yourself, or "SlimeVR Contributors"